test/org/zaproxy/zap/extension/pscanrulesAlpha/FeaturePolicyScannerUnitTest.java

```diff
@@ -0,0 +1,121 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2019 The ZAP development team
```

Comment:
Should have caps on Development Team. I’m sure this is a copy/paste thing, we’re working on addressing it 🙂

Reply:
Resolved with dd615238d8beff203798389b18613cc05af850f5